### ISSUE TITLE
[Bug] Only system fonts are used in theme

### ISSUE THREAD

**Issue by haw0k** [creativetimofficial/ct-purity-ui-dashboard-pro#3]
### Version
2.0.1
### Reproduction link
https://demos.creative-tim.com/purity-ui-dashboard/
### Operating System
elementary OS 6.1 (based on Ubuntu 20.04.5 LTS)
### Device
PC
### Browser & Version
Firefox 104.0.1 (64-bit)
### Steps to reproduce

    Open link with Purity UI live demo in Firefox at OS without preinstalled fonts Roboto, Helvetica etc.
    Open Firefox developers tools > Inspector > Fonts > All Fonts on Page
    Check the fonts list - we can see only system fonts.

next steps reproduces my attempt to adding the Roboto font to local copy of Purity UI theme:

    $ git clone https://github.com/creativetimofficial/purity-ui-dashboard
    $ cd purity-ui-dashboard/
    $ yarn
    $ yarn start
    Open http://localhost:3000/ in Firefox at OS without preinstalled fonts Roboto, Helvetica etc.
    Open Firefox developers tools > Inspector > Fonts > All Fonts on Page
    Check the fonts list

### What is expected?
Expect to view 'Roboto' font because src/theme/foundations/fonts.js contains following:
export const font = {
body: 'Roboto'
};
### What is actually happening?
we can see only preinstalled system fonts: TeX Gyre Heros, Twemoji Mozilla at my OS

---
### Solution
fix font support (@fontsource/open-sans, @fontsource/raleway, @fontsource/roboto)
### Additional comments
The same problem is observed in the free version of purity-dashboard-react 2.0.2

**Comment by simmmpleweb** [comment]
Hello, @haw0k  ! Thanks for using our products. Problem was solved just as on the Free version: I deleted `fonts.js` entirely, and now if you import the fonts in index.html, then set the font in `src/theme/styles.js`, you will no longer face this problem. Regards, Vlad.
<img width="926" alt="Screenshot 2022-09-08 at 19 19 54" src="https://user-images.githubusercontent.com/90040741/189179747-6dc28b46-f514-4c5e-80c2-55701ece2865.png">
<img width="1446" alt="Screenshot 2022-09-08 at 19 19 32" src="https://user-images.githubusercontent.com/90040741/189179749-24e20d67-e4f9-48e2-9b29-83363991565d.png">
